aten/src/ATen/FuncTorchTLS.cpp

```diff
@@ -0,0 +1,27 @@
+#include <ATen/FuncTorchTLS.h>
+
+namespace at { namespace functorch {
+
+thread_local std::shared_ptr<FuncTorchTLSBase> kFuncTorchTLS = nullptr;
```

Comment:
nit you can put this in a private namespace to make sure it doesn't end up being used in other places?

Reply:
Sure